### DISCUSSION TITLE
第2节课作业(14班)

### DISCUSSION THREAD

**Discussion by NewDaNew** [InternLM/Tutorial#121] [作业]
第二节课作业哦～

**Comment by Xinyusei** [comment]
[作业](https://github.com/Xinyusei/InternLM/blob/main/Notes2/Homework-2.md)
[笔记](https://github.com/Xinyusei/InternLM/blob/main/Notes2/Notes-2.md)

**Comment by zenith779** [comment]
https://www.wolai.com/jinagzk2018/omxNsNXche96ieHUpXG5A4

**Comment by ZZfive** [comment]
https://sincere-papyrus-971.notion.site/845a8a272ed64fb39cb6e630352276c8?pvs=4

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/uzlem7r1p63gpqbi?singleDoc# 《HelloInternLM》

**Comment by Nobitaxi** [comment]
https://blog.csdn.net/weixin_44288092/article/details/135439455?spm=1001.2014.3001.5501

**Comment by lizihe11** [comment]
https://gitee.com/kiddylzh/intern-lm-tutorial-note/blob/master/task/second%20task.md

**Comment by Mercuryoo** [comment]
https://t3d0h0gdxm.feishu.cn/docx/PonxdAESwozGoIxJh2mcgoainjh?from=from_copylink

**Comment by sunsearain** [comment]
https://blog.csdn.net/weixin_47233613/article/details/135463739?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135463739%22%2C%22source%22%3A%22weixin_47233613%22%7D

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135469619?spm=1001.2014.3001.5501

**Comment by wakefuldreamer** [comment]
https://www.yuque.com/gggggenius/fmhsbr/elcg9syack620r84?singleDoc# 《第二课作业》

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135469619?spm=1001.2014.3001.5501（作业+笔记）

**Comment by xilimala** [comment]
作业：https://leaf-fennel-588.notion.site/4ff7828045cf4019bcccb56eb447566c?pvs=4
笔记：https://leaf-fennel-588.notion.site/ada5446240d04d00b8bc6d14b60a714a?pvs=4

**Comment by tianyabanbu** [comment]
https://qiz21j4rsiq.feishu.cn/docx/O8SndkEaQoV0RPxDeGrcbdxFn5n?from=from_copylink

**Comment by ucas-ai** [comment]
https://blog.csdn.net/u012468965/article/details/135593813?spm=1001.2014.3001.5502
